src/routes.jsx

```diff
@@ -69,5 +70,12 @@ export const routes = [
 ];
 
 export default function(initialProps = {}, locale) {
-  return <App initialProps={initialProps} locale={locale} />;
+  return (
+    <ExperimentsContext.Provider
```

Comment:
Bruken av ExperimentsContext er litt forvirrende. Nå brukes samme Context til to foskjellige providers, det virker ikke helt riktig. Ville definert en egen context her og importert den i stedet